eland/ml/pytorch/transformers.py

```diff
@@ -756,7 +775,7 @@ def _create_config(
             ),
         )
 
-    def _create_traceable_model(self) -> TraceableModel:
+    def _create_traceable_model(self) -> Tuple[Any, TraceableModel]:
```

Comment:
I find this change introduces some complexity that could maybe be encapsulated better by just adding the `transformer_model` to `_TransformerTraceableModel`. We do this with the tokenizer, which gets saved as a property of that class. It would make it cleaner to do that with the underlying HF model, and would mean we don't need to return tuples here. If it doesn't make sense to return a `TraceableModel` then maybe it's worth switching to return `_TransformerTraceableModel`.